### DISCUSSION TITLE
Selected For GitHub Grants for Open Source.

### DISCUSSION THREAD

**Discussion by anuraghazra** [anuraghazra/github-readme-stats#1310] [Announcement]
### 🎉 I'm super exited to announce that I've been selected for GitHub Grants for Open Source.

![grant-oss-github-banner](https://user-images.githubusercontent.com/35374649/133122702-a360e0f1-8dbc-4ac4-8b8a-65e8403c22a0.png)

Huge thanks to [GitHubIndia](https://twitter.com/GitHubIndia) & [GitHub](https://twitter.com/github) for supporting open source developers of India and help us build open source software and make an impact for India and the world. ❤️ 

I can't express my excitement and joy. 
Congratulations to other recipients as well. 

- Announcement Post: https://github.blog/2021-09-12-recipients-open-source-grants-github-sponsors-india
- Announcement Video: https://twitter.com/GitHubIndia/status/1437352013814136832
- GitHub India Grants: https://github.com/github/india/blob/main/Open%20Source/GRANTS.md

------ 

## Thank You Contributors

And to thank github-readme-stats contributors & friends who regularly help me maintain this project I'll one-time sponsor them via github sponsors. (within this week)

Nominees are: 

- @filiptronicek - Thank you for helping from the start
- @nathanchu - Thank you for amazing contributions and ideas
- @florianbussmann - Thank you for maintaining and triaging various issues

**Comment by filiptronicek** [comment]
Thanks for the nomination, @anuraghazra! It's been really great seeing this project grow, and to see **you** grow, as well.

**Reply by anuraghazra** [reply]
Welcome & Thanks filip! 🥂

**Comment by kahy9** [comment]
@filiptronicek = OpenSource, he´s not the oldest one, but he's done a lot for the world of OpenSource! ❤

**Reply by filiptronicek** [reply]
Awww, you shouldn't have 😛❤️

**Reply by krystxf** [reply]
Hope Filip's DNA will be opensource one time
